software/controller/lib/core/sensors.cpp

```diff
@@ -28,6 +28,7 @@ static DebugFloat dbg_dp_exhale("dp_exhale", "Exhale diff pressure, cmH2O");
 static DebugFloat dbg_pressure("pressure", "Patient pressure, cmH2O");
 static DebugFloat dbg_flow_inhale("flow_inhale", "Inhale flow rate, cc/sec");
 static DebugFloat dbg_flow_exhale("flow_exhale", "Exhale flow rate, cc/sec");
+static DebugFloat dbg_fio2("fio2", "Fraction of inspired oxygen, [0,100%]");
```

Comment:
I made a mistake here, this should be [0.0, 1.0] not [0, 100%]

Reply:
Sorry for taking so long to reply, I noticed the original mistake, and thought having the value between 0 and 100% would be easier to follow, so I changed the setting line to: dbg_fio2.Set(fio2 * 100.0f);
I'll change it back to 0-1 if you prefer it that way, as the difference is very small in my eyes.

Reply:
I think it's better to use 0.0-1.0 as when we plot this we track and compare the setpoint (params.fio2), current reading (sensor_readings.fio2), and valve command and these are all defined as 0.0-1.0 in the code.